src/components/MyPage/project-result/ProjectResultPage.tsx

```
@@ -10,18 +10,25 @@ import SettlementTab from './tabs/SettlementTab';
 const ProjectResultPage = () => {
-  const { id } = useParams<{ id: string }>();
+  const { projectId } = useParams();
 
+  if (!projectId || isNaN(Number(projectId))) {
+    return <div>잘못된 접근입니다.</div>;
+  }
+
+  const numericProjectId = Number(projectId);
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
   const [activeTab, setActiveTab] = useState<ProjectTab>('dashboard');
```

Biome (2.3.14)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)